.github/workflows/review_pull_request.yml

```diff
@@ -46,6 +46,16 @@ jobs:
 
       # template-compose
 
+      - name: Set up google-services.json on template-compose
+        env:
+          GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
+          GOOGLE_SERVICES_JSON_STAGING: ${{ secrets.GOOGLE_SERVICES_JSON_STAGING }}
+        run: |
+          mkdir -p template-compose/app
+          echo $GOOGLE_SERVICES_JSON > template-compose/app/google-services.json
+          mkdir -p template-compose/app/src/staging
+          echo $GOOGLE_SERVICES_JSON_STAGING > template-compose/app/src/staging/google-services.json
```

Comment:
this workflow needs staging credential only 🙏

Reply:
@luongvo It doesn't work yet. Please help me check the script. 😢

Reply:
please contact cc @ryan-conway for help or permission

Reply:
@luongvo Didn't you mean this env?

![Screenshot 2566-07-18 at 6 14 17 PM](https://github.com/nimblehq/android-templates/assets/12026942/f4e9755a-f606-46d6-8e9c-94cbf6359999)

Reply:
@lydiasama So the environment name is `template-compose`? I don't see you set it to this workflow https://github.com/nimblehq/flutter-templates/blob/13bd90464e1f0dd9ed915ca495c7032eea31880a/.github/workflows/android_deploy_staging.yml#L13C25-L13C25

Reply:
@lydiasama I think it’s better to have 2 env here: `staging` and `production`, instead of by project name, as we will support `compose only`.

Reply:
@luongvo Ahh. 🙈  I thought I needed to define it somewhere but don't know where I should add it. Many thanks!

Reply:
> this workflow needs staging credential only, please correct me if I'm wrong 🙏

@luongvo  It requires production too. :

![SCR-25660719-jqql](https://github.com/nimblehq/android-templates/assets/12026942/09ef62b1-0601-4db9-b2ac-0624f49a4097)

Reply:
@lydiasama ok, we're using `./gradlew lint` instead of only debug `./gradlew lintDebug`